apps/www/registry/new-york/ui/select.tsx

```diff
@@ -3,6 +3,7 @@
 import * as React from "react"
 import { CaretSortIcon, CheckIcon } from "@radix-ui/react-icons"
 import * as SelectPrimitive from "@radix-ui/react-select"
+import { ChevronDown, ChevronUp } from "lucide-react"
```

Comment:
For `new-york`, let's use icons from `@radix-ui/react-icons`

Reply:
Updated as per request & commited.